js/sign/src/utils/utils.ts

```diff
@@ -34,3 +34,20 @@ export function getRawPublicKey(publicKey: crypto.KeyObject) {
     publicKey.export({ type: 'spki', format: 'der' }).slice(-32)
   );
 }
+
+export function isValidEd25519Key(
```

Comment:
nit: Rename to `checkIsValidEd25519Key` and add a comment to the method saying something along the lines of "will throw if the key is not a valid Ed2559 key of the specified type".